digiroad2-api-oth/src/test/scala/fi/liikennevirasto/digiroad2/dataexport/CsvDataExporterSpec.scala

```diff
@@ -0,0 +1,69 @@
+package fi.liikennevirasto.digiroad2.dataexport
```

Comment:
I don't know why you are testing this createStandardCSVData. There is no other place that method is being used except here

Reply:
It is a "generic" class for standard csv creation.
So I think it the test make all sense.

Reply:
That's true, I remember now we discussed that, my bad.